www/src/pages/en/folder-structure.md

```diff
@@ -0,0 +1,240 @@
+
+### `src/server/common/get-server-auth-session.ts`
+
+The `get-server-auth-session.ts` file is used to get the NextAuth.js session on the server-side. See [NextAuth.js usage](usage/next-auth#usage-with-trpc) for more information.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+### `src/server/db/client.ts`
+
+The `client.ts` file is used to instantiate the Prisma client at global scope. See [Prisma usage](usage/prisma#prisma-client) for more information.
+
+<sub>(with Prisma)</sub>
+
+## `src/server/trpc`
+
+The `trpc` folder contains the tRPC server-side code.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/context.ts`
+
+The `context.ts` file is used to create the context used in tRPC requests. See [tRPC usage](usage/trpc#-servertrpccontextts) for more information.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/trpc.ts`
+
+The `trpc.ts` file is used to export procedure helpers. See [tRPC usage](usage/trpc#-servertrpctrpcts) for more information.
+
+<sub>(with tRPC)</sub>
+
+## `src/server/trpc/router`
+
+The `router` folder contains the tRPC routers.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/router/_app.ts`
+
+The `_app.ts` file is used to merge tRPC routers and export them as a single router, as well as the type definitions. See [tRPC usage](usage/trpc#-servertrpcrouterts) for more information.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/router/auth.ts`
+
+The `auth.ts` file is an example tRPC router utilizing the `protectedProcedure` helper to demonstrate how to protect a tRPC route with NextAuth.js.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+### `src/server/trpc/router/example.ts`
+
+The `example.ts` file is an example tRPC router utilizing the `publicProcedure` helper to demonstrate how to create a public tRPC route.
+
+<sub>(with tRPC)</sub>
+
+## `src/styles`
+
+The `styles` folder contains the global styles of the application.
+
+<sub>(with Tailwind CSS)</sub>
+
+## `src/types`
+
+The `types` folder is used to store reused types or type declarations.
+
+<sub>(with NextAuth.js)</sub>
+
+### `src/types/next-auth.d.ts`
+
+The `next-auth.d.ts` file is used to extend the NextAuth default session type to include the user ID. See [NextAuth.js usage](usage/next-auth#inclusion-of-userid-on-the-session) for more information.
+
+<sub>(with NextAuth.js)</sub>
+
+## `src/utils`
+
+The `utils` folder is used to store commonly re-used utility functions.
+
+<sub>(with tRPC)</sub>
+
+### `src/utils/trpc.ts`
+
+The `trpc.ts` file is the front-end entry point to tRPC. See [tRPC usage](usage/trpc#-utilstrpcts) for more information.
+
+<sub>(with tRPC)</sub>
+
+## `.env`
+
+The `.env` file is used to store environment variables. See [Environment variables](usage/env-variables) for more information. This file should **not** be commited to git history.
+
+## `.env.example`
+
+The `.env.example` file shows example environment variables based on the chosen libraries. This file should be commited to git history.
```

Comment:
This file is called .env-example currently

Reply:
should change that then, .env.example is better imo

Reply:
> should change that then, .env.example is better imo

Are we doing this before the release

Reply:
yea i agree calling it .env.example is better